rasterio/__init__.py

```diff
@@ -50,135 +51,13 @@ def open(
         dtype=None,
         nodata=None,
         **kwargs):
-    """Open file at ``path`` in ``mode`` "r" (read), "r+" (read/write),
-    or "w" (write) and return a ``Reader`` or ``Updater`` object.
+    """See Env.open()"""
+    env = Env()
```

Comment:
@sgillies I think this needs to be `env = kwargs.pop('env', Env())` to support (should we choose to keep this behavior):

```python
with rio.Env() as env:
    with rio.open('tests/data/RGB.byte.tif', env=env) as src:
        print(src._env.options)
Traceback (most recent call last):
  File "buh.py", line 5, in <module>
    with rio.open('tests/data/RGB.byte.tif', env=env) as src:
  File "/Users/wursterk/code/rasterio/rasterio/__init__.py", line 58, in open
    nodata=nodata, env=env, **kwargs)
TypeError: open() got multiple values for keyword argument 'env'
```